test/e2e/app-dir/rsc-webpack-loader/rsc-webpack-loader.test.ts

```diff
@@ -1,4 +1,6 @@
 import { nextTestSetup } from 'e2e-utils'
+
+// Skip as Turbopack doesn't support webpack loaders.
```

Comment:
But we do support webpack loaders? Maybe we don't support it in this specific usage?

Reply:
I've updated the comment:
```
// Skip as Turbopack doesn't support the `!=!` Webpack syntax
```